packages/base/src/Tab/Tab.tsx

```diff
@@ -162,7 +163,7 @@ export const TabBase: RneFunctionComponent<TabProps> = ({
     }
     const inputRange = Array.from(Array(countItems + 1).keys());
     const outputRange = tabItemPositions.current.map(
-      ({ position }) => position
+      ({ position }) => I18nManager.isRTL ? -position : position
```

Comment:
Hey @PoojaBabusingh, can you fix the linting issue here? Thanks for raising the PR. 😄